src/ol/format/GeoJSON.js

```diff
@@ -17,6 +17,21 @@ import Point from '../geom/Point.js';
 import Polygon from '../geom/Polygon.js';
 import {assign, isEmpty} from '../obj.js';
 import {get as getProjection} from '../proj.js';
+import GeometryType from '../geom/GeometryType.js';
+
+/**
+ * @typedef {import("geojson").GeoJSON} GeoJSONObject
```

Comment:
I was surprised to find that `import("@types/geojson")` does not work and that this is the way to import packages from the `@types` scope.  Unfortunate that this makes it so people cannot import from arbitrary packages.

Reply:
That's interesting indeed.